### PR TITLE
Implement `from_env_var` for `TestRunner`

### PR DESCRIPTION
**Pull request by pacmanifold**

### CODE REVIEW

File: src/artifact/mod.rs [hunk 1]
```diff
@@ -114,4 +114,6 @@ impl Artifact {
             } => todo!(),
         }
     }
+
+
```
Comment (apollo-sturdy) [inline src/artifact/mod.rs:118]: rustfmt?

File: src/test_runner.rs [hunk 2]
```diff
@@ -16,6 +19,8 @@ use osmosis_test_tube::OsmosisTestApp;
 /// An enum with concrete types implementing the Runner trait. We specify these here because the
 /// Runner trait is not object safe, and we want to be able to run tests against different types of
 /// runners.
+#[derive(strum::EnumVariantNames)]
+#[strum(serialize_all = "kebab_case")]
```
Comment (apollo-sturdy) [inline src/test_runner.rs:23]: what does this do?

File: Cargo.toml [hunk 3]
```diff
@@ -34,21 +34,7 @@ rpc-runner = ["testcontainers", "tonic", "tokio", "cosmos-sdk-proto"]
...
-  "astroport-generator",
-  "astroport-maker",
-  "astroport-pair-stable",
-  "astroport-pair",
-  "astroport-router",
-  "astroport-staking",
-  "astroport-vesting",
-  "astroport-whitelist",
-]
+astroport-multi-test = ["multi-test", "astroport", "astroport-token", "astroport-native-coin-registry", "astroport-factory", "astroport-generator", "astroport-maker", "astroport-pair-stable", "astroport-pair", "astroport-router", "astroport-staking", "astroport-vesting", "astroport-whitelist"]
```
Comment (apollo-sturdy) [inline Cargo.toml:37]: why destroy pretty fmt?

File: src/test_runner.rs (hunk 2 above)
Comment (pacmanifold) [reply src/test_runner.rs:23]: generates a constant `TestRunner::VARIANTS` that can be used in the tests. If we add a new variant without also updating the relevant tests they will fail

**Pull request merged**